### PR TITLE
docs(storybook): add configuration and basic story

### PR DESCRIPTION
**Pull request by SimonFinney**
## Changelog

**New**

- Add Storybook environment and SCSS configuration to `core`
- Add basic `ComboButton` story for Security
- Add template stories for other packages - `automation`, `cdai`, `common-services`
- Move React dependency up to the root

### CODE REVIEW

File: packages/cloud-paks/src/ComboButton.stories.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+export const Default = () => (
+  <ComboButton>
+    <ComboButtonItem>ComboButtonItem 1</ComboButtonItem>
+    <ComboButtonItem>ComboButtonItem 2</ComboButtonItem>
+  </ComboButton>
+);
+
+export default {
+  title: 'IBM Security/ComboButton',
```
Comment (dbrugger) [inline packages/cloud-paks/src/ComboButton.stories.js:21]:
```suggestion
  title: 'Cloud Paks/ComboButton',
```

**Comment by SimonFinney** [comment]
Thanks @dbrugger ! This is ready for re-review as per our conversation earlier

**Comment by SimonFinney** [comment]
@joshblack It appears the status check is stalling here - would you know why this might be the case?

**Comment by joshblack** [comment]
@SimonFinney should be good now! Forgot to add the `pull_request` event 🤦‍♂️ haha. Let me know if you run into any other problems!

**Pull request merged**